changes/1569.feature.rst

```diff
@@ -0,0 +1,3 @@
+Corrected line length
+Changed definition listed for cache and toga properties
+Updated formating to use ReST styles
```

Comment:
The change note is the single line that will be included in the release notes relating to this change. It's not a summary of code changes; it's a user-facing description of what has changed. The `feature` means it's a major improvement in the framework. "Corrected line length" won't be a new feature in Toga v0.3.2 :-)

This is a documentation update; so it should be `.doc.rst`. See [the contribution guide](https://toga.readthedocs.io/en/stable/how-to/contribute-code.html#add-change-information-for-release-notes) for details. 
```suggestion
Documentation for application paths was added.
```